sample-config/test-config.json

```diff
@@ -5,7 +5,8 @@
     "deviceTestType": "SELENIUM_NODE",
     "type": "BROWSER",
     "executionConfiguration": {
-      "videoCapture": true
+      "videoCapture": true,
+      "testArgs": "--some --test --arguments"
```

Comment:
Perhaps we can use the same example here as in the arg description i.e. `-spec tests/login-test.ts`